Apps/Sandcastle/gallery/CZML Point Time Dynamic.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+  <meta name="description" content="CZML Point Time Dynamic">
+  <meta name="cesium-sandcastle-labels" content="CZML">
+  <title>Cesium Demo</title>
+  <script type="text/javascript" src="../Sandcastle-header.js"></script>
+  <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.9/require.js"></script>
+  <script type="text/javascript">
+  require.config({
+    baseUrl : '../../../Source',
+    waitSeconds : 60
+  });
+  </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+@import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+
+<script id="cesium_sandcastle_script">
+
+function startup(Cesium) {
+  "use strict";
+  //Sandcastle_Begin
+  var czml = [
+    {
+      "id" : "document",
+      "name" : "CZML Point Time Dynamic",
+      "version" : "1.0"
+    }, {
+      "id": "point",
+      "availability":"2012-08-04T16:00:00Z/2012-08-04T17:00:00Z",
+      "position": {
+        "referenceFrame": "INERTIAL",
```

Comment:
Do not use points defined in degrees in the Inertial reference frame; instead, just use a satellite from simple.czml to show the Inertial frame.